### PR TITLE
[Laravel-Emojis] Add more facial emojis (emoticons)

### PR DESCRIPTION
**Pull request by vdugeri**

### CODE REVIEW

File: src/Emojis/emoji.php [hunk 1]
```diff
@@ -30,5 +30,38 @@
...
     "kissing_closed_eyes"       => "\u{1F61A}",
     "relaxed"                   => "\u{263A}",
-
-];
\ No newline at end of file
+    "relieved"                  => "\u{1F60C}",
+    "stuck_out_tongue"          => "\u{1F61C}",
+    "stuck_out_tongue_closed_eyes" => "\u{1F61D}",
+    "disappointed"              => "\u{1F61E}",
+    "angry"                     => "\u{1F620}",
```
Comment (unicodeveloper) [inline src/Emojis/emoji.php:49]: Please can you ensure they are ordered appropriately via the unicode value? You actually ordered a greater part of it but I think "relaxed" should be the last, "relieved" should come just after  "yum" .

**Comment by unicodeveloper** [comment]
Please, what's the latest on this @andela-vdugeri ?

File: src/Emojis/emoji.php (hunk 1 above)
Comment (vdugeri) [inline src/Emojis/emoji.php:49]: Sorry about the late reply, jus seeing this now. Will work on it ASAP

**Comment by unicodeveloper** [comment]
Please rebase @andela-vdugeri

**Comment by unicodeveloper** [comment]
@andela-vdugeri Just a subtle reminder. Please can you rebase, so that I can merge this in 😊

**Comment by unicodeveloper** [comment]
Great @andela-vdugeri

**Pull request merged**